code/obj/item/cigarette.dm

```diff
@@ -2,6 +2,136 @@
 /* ==================================================== */
 /* -------------------- Cigarettes -------------------- */
 /* ==================================================== */
+/obj/item/clothing/mask/cig_bundle
+	name = "bundle of cigarettes"
+	icon = 'icons/obj/items/cigarettes.dmi'
+	wear_image_icon = 'icons/mob/mask.dmi'
+	icon_state = "cig_bundle-2"
+	uses_multiple_icon_states = 1
+	item_state = "cig_bundle"
+	var/list/cig_list = list()
+	var/const/max_cigs = 21
```

Comment:
Is there a technical reason for making this a `const`?

Reply:
Not particularly, I don't think that const has any performance implications, but this number shouldn't be changed since I hard code different icon states up to 21 cigs.